src/config.ts

```diff
@@ -1,97 +1,51 @@
 /**
  * config.ts
- * 
- * Configuration loader for the Launchtube plugin.
- * Manages plugin settings including relayer IDs, network configuration, and RPC endpoints.
+ *
+ * Environment-driven configuration for the Launchtube plugin.
+ * Removes legacy config.json usage and relies on env vars.
  */
 
-import fs from 'fs';
-import path from 'path';
 import { Networks } from '@stellar/stellar-sdk';
+import { pluginError } from '@openzeppelin/relayer-sdk';
 
 export interface LaunchtubeConfig {
   fundRelayerId: string;
-  sequenceRelayerIds: string[];
-  maxFee: number;
   network: 'testnet' | 'mainnet';
   rpcUrl: string;
 }
 
+function requireEnv(name: string): string {
+  const v = process.env[name];
+  if (!v || v.trim() === '') {
+    throw pluginError(`Missing required environment variable: ${name}`, {
+      code: 'CONFIG_MISSING',
+      status: 500,
+      details: { name },
+    });
+  }
+  return v.trim();
+}
+
 /**
- * Load configuration from config.json
+ * Load configuration from environment variables
  */
 export function loadConfig(): LaunchtubeConfig {
-  // Search for config.json in multiple locations
-  const possiblePaths = [
-    // 1. Check plugins/launchtube directory (when used in relayer)
-    path.join(process.cwd(), 'plugins', 'launchtube', 'config.json'),
-    // 2. Check plugins directory 
-    path.join(process.cwd(), 'plugins', 'config.json'),
-    // 3. Current working directory
-    path.join(process.cwd(), 'config.json'),
-    // 4. Plugin's own directory (for local development)
-    path.join(__dirname, '..', 'config.json'),
-  ];
-
-  let configPath: string | undefined;
-  for (const possiblePath of possiblePaths) {
-    if (fs.existsSync(possiblePath)) {
-      configPath = possiblePath;
-      console.log(`Loading launchtube config from: ${configPath}`);
-      break;
-    }
+  const networkRaw = requireEnv('STELLAR_NETWORK').toLowerCase();
+  if (networkRaw !== 'testnet' && networkRaw !== 'mainnet') {
+    throw pluginError('STELLAR_NETWORK must be "testnet" or "mainnet"', {
+      code: 'UNSUPPORTED_NETWORK',
+      status: 400,
+    });
   }
 
-  if (!configPath) {
-    throw new Error(
-      `Configuration file not found. Searched in:\n` +
-      possiblePaths.map(p => `  - ${p}`).join('\n') +
-      `\n\nPlease create a config.json file in one of these locations.`
-    );
-  }
-
-  try {
-    const configData = fs.readFileSync(configPath, 'utf8');
-    const config = JSON.parse(configData);
-
-    // Validate required fields
-    if (!config.fundRelayerId) {
-      throw new Error('Missing fundRelayerId in config');
-    }
+  const rpcUrl = requireEnv('SOROBAN_RPC_URL');
```

Comment:
Future improvement, we should expose rpc method from useRelayer so plugins could reuse rpc urls already set and reuse retry and failover logic we have there.